<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>823</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>МАИ Раcчет деформации</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Изображение до деформации</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Изображение после деформации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="before_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>621</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать изображение до деформации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="after_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>670</y>
      <width>621</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать изображение после деформации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>710</y>
      <width>931</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет векторов</string>
    </property>
   </widget>
   <widget class="QLabel" name="before_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>620</width>
      <height>620</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="after_img">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>620</width>
      <height>620</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="choose_method">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>lucas_kanade [не работает]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>farneback</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>farneback2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>321</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите метод</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_get_before_img"/>
    <addaction name="action_get_after_img"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_get_before_img">
   <property name="text">
    <string>Выбрать изображение до деформации [не работает]</string>
   </property>
  </action>
  <action name="action_get_after_img">
   <property name="text">
    <string>Выбрать изображение после деформации [не работает]</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход [не работает]</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
